src/libraries/System.Security.AccessControl/src/System/Security/AccessControl/ACE.cs

```diff
@@ -328,16 +323,7 @@ public static GenericAce CreateFromBinaryForm(byte[] binaryForm, int offset)
                 type == AceType.SystemAuditCallbackObject ||
                 type == AceType.SystemAlarmCallbackObject)
             {
-                AceQualifier qualifier;
-                int accessMask;
-                SecurityIdentifier sid;
-                ObjectAceFlags objectFlags;
-                Guid objectAceType;
-                Guid inheritedObjectAceType;
-                bool isCallback;
-                byte[] opaque;
-
-                if (true == ObjectAce.ParseBinaryForm(binaryForm, offset, out qualifier, out accessMask, out sid, out objectFlags, out objectAceType, out inheritedObjectAceType, out isCallback, out opaque))
+                if (ObjectAce.ParseBinaryForm(binaryForm, offset, out AceQualifier qualifier, out int accessMask, out SecurityIdentifier? sid, out ObjectAceFlags objectFlags, out Guid objectAceType, out Guid inheritedObjectAceType, out bool isCallback, out byte[]? opaque))
```

Comment:
Moving these variable declarations inline has taken a long line and made it a very long line.  Consider chopping, or just undo moving the variable declarations.

Reply:
Yeah, will chop the line